api/watchlist/watchlist-bridgesModel.js

```diff
@@ -0,0 +1,74 @@
+const db = require('../../data/db-config');
+
+async function findWatchlist(id) {
+  let list = await db('watchlist').where('profile_id', id);
+  const bridges = await findBridges(id);
+  console.log(bridges);
+  // console.log('list: ', list, 'bridges', bridges);
+  list = {
+    ...list[0],
+    locations: bridges,
+  };
+
+  return list;
+}
+
+async function findBridges(id) {
+  let bridge_array = [];
+  const bridges = await db('watchlist_bridges')
+    .where('list_id', id)
+    .select('project');
+
+  bridges.forEach((bridge) => {
+    console.log(bridge);
+    bridge_array.push(bridge.project);
+  });
+
+  return bridge_array;
+}
+
+const addWatchlist = async (title, profile_id, notes, bridge_array) => {
+  // console.log(title, profile_id, notes, bridge_array);
```

Comment:
I would probably remove console logs like this

Reply:
Thanks, I've cleaned them up in the latest version